Framework/PythonInterface/plugins/algorithms/VesuvioPeakPrediction.py

```diff
@@ -107,6 +107,23 @@ def PyExec(self):
 
                 vesuvio_params.addRow([temp, self._atomic_mass, self._debye_temp, kinetic, rms_momentum, rms_disp])
 
+        if self._model == "Classical":
+            vesuvio_params.addColumn("float", "RMS Momentum(A-1)")
+            vesuvio_params.addColumn("float", "Kinetic Energy(mEV)")
+
+            for temp in self._temperature:
+                if temp == 0:
+                    temp = MIN_TEMPERATURE
+
+                temp_mev = temp / K_IN_MEV
+                # Reduced Planck constant in units sqrt(mev * amu) / A-1
+                h_bar = 2.04434
```

Comment:
This constant seems like it would be better suited being at the top of the file with the others.